tests/browser/http/test_content_disposition.py

```diff
@@ -20,11 +20,11 @@
 """Tests for qutebrowser.browser.http.parse_content_disposition."""
 
 import os
-import unittest
 import logging
 
```

Comment:
Haha :smile: 

In the **split** view there is a blank line between ``import logging`` and ``import pytest``.